### ISSUE TITLE
Losses used in training

### ISSUE THREAD

**Issue by Maxcharm** [UX-Decoder/Segment-Everything-Everywhere-All-At-Once#35]
Hi! Wonderful project you have here :)
just wondering if you can provide us with all the losses you use in the training step, since it is not mentioned in the paper.
thanks a lot!

**Comment by MaureenZOU** [comment]
Here is the loss function: L =αLc_CE_pano + βLm_BCE_pano + γLm_DICE_pano + aLc_CE_ref + bLm_BCE_ref
+cLm_DICE_ref + aLc_CE_iseg + bLm_BCE_iseg + cLm_DICE_iseg
α,β,γ,a,b,c=2,5,5,0.2,2,2
